test/lib/testLocalStore.js

```diff
@@ -49,16 +48,19 @@ describe('localStore [node.js version for testing]', function() {
   });
 
   describe('localStore.getItem', function() {
-    var data = {blocks: {mine: ['a', 'b'], yours: [1,2,3]}};
-    localStore = localStore(data);
+    var data = 'awesome_data';
```

Comment:
Hmm, and I have a feeling that authentication token is not being refreshed as expected, leading to uploads failing if you leave the Uploader open for a long time:
https://trello.com/c/yQx6JyQB. So we can revisit this when we finally get around to fixing that bug.